Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebar.razor.cs

```diff
@@ -42,16 +42,24 @@ public partial class HxSidebar : ComponentBase
 		/// </summary>
 		[Parameter] public string Id { get; set; } = "hx-" + Guid.NewGuid().ToString("N");
 
-		protected internal bool Collapsed { get; set; } = false;
+		/// <summary>
+		/// Indicates whether the <see cref="HxSidebar"/> is collapsed, can be used to alter the state (expand or collapse the sidebar).
+		/// </summary>
+		[Parameter] public bool Collapsed { get; set; } = false;
+		/// <summary>
+		/// Fires when the sidebar is expanded or collapsed.
+		/// </summary>
+		[Parameter] public EventCallback<bool> CollapsedChanged { get; set; }
```

Comment:
Follow the `protected virtual Task InvokeCollapsedChangedAsync(..)` pattern to support extensibility seam in derived components. See other components implementation.